src/database/index.js

```diff
@@ -1,6 +1,6 @@
 'use strict';
 
-const nconf = require('nconf');
```

Comment:
this change is causing the tests to fail; i think you need to undo it to make them pass